src/views/Designs.vue

```diff
@@ -64,142 +161,153 @@
               </tr>
             </template>
             <template v-slot:expand="{ item: design }">
-              <tr>
-                <td>
-                  <v-checkbox hide-details class="hidden"></v-checkbox>
-                </td>
-                <td align="rigth" width="20%">Name</td>
-                <td align="rigth" width="15%">Organism</td>
-                <td align="rigth" width="15%">Model</td>
-                <td align="rigth" width="15%">
-                  <div class="link-list">
-                    <div
-                      v-for="(reactionKnockin, index) in design.design
-                        .reaction_knockins"
-                      :key="index"
-                    >
-                      <div v-if="index < 10">
-                        <a
-                          :href="
-                            reactionLink(reactionKnockin, design.method, true)
-                          "
-                          class="link"
-                          target="_blank"
-                        >
-                          {{ getReactionId(reactionKnockin, design.method) }}
-                        </a>
-                      </div>
+              <v-container style="padding:0;" class="elevation-1">
+                <v-layout justify-end>
+                  <v-flex
+                    style="width:55%; padding:0 24px; box-sizing: border-box;"
+                  ></v-flex>
+                  <v-flex
+                    style="width:15%; padding:0 24px; box-sizing: border-box;"
+                  >
+                    <div class="link-list">
                       <div
-                        v-if="index >= 10"
-                        :hidden="!showAllReactionKnockins"
+                        v-for="(reactionKnockin, index) in design.design
+                          .reaction_knockins"
+                        :key="index"
                       >
-                        <a
-                          :href="
-                            reactionLink(reactionKnockin, design.method, true)
-                          "
-                          class="link"
-                          target="_blank"
+                        <div v-if="index < 10">
+                          <a
+                            :href="
+                              reactionLink(reactionKnockin, design.method, true)
```

Comment:
This is fine for now, in a future refactor I would make this an added property on the `reactionKnockin` itself.